src/fs_event_wrap.cc

```diff
@@ -105,22 +127,19 @@ void FSEventWrap::New(const FunctionCallbackInfo<Value>& args) {
   new FSEventWrap(env, args.This());
 }
 
-
+// wrap.start(filename, persistent, recursive, encoding)
```

Comment:
this line is commented out. Can you remove it?

Reply:
@mcollina It's the signature of this method...probably less obvious than I thought

Reply:
I think it is nice to have the signature. I am not sure what the convention for this is in C++ though. Something similar to JSDoc might be nicer?

Reply:
This is more about the positions, the types are pretty obvious given the assertions below.

Reply:
I do not have a strong opinion... hm... maybe change `wrap.start` to: `Arguments: `?